packages/yoga/src/Input/web/Fieldset.jsx

```diff
@@ -33,12 +33,14 @@ const Fieldset = styled.fieldset`
 
     ${value && !error ? `border-color: ${input.border.color.typed};` : ''}
 
-    &:hover, &:focus-within {
-      border-color: ${
-        error ? colors.feedback.attention[1] : input.border.color.typed
-      };
-
-      ${disabled ? `border-color: ${colors.elements.lineAndBorders};` : ''}
+    ${disabled && `border-color: ${colors.elements.lineAndBorders};`}
```

Comment:
```suggestion
    ${disabled ? `border-color: ${colors.elements.lineAndBorders};` : ''}
```

That way we don't have `false` on our generated CSS. (As we can see on the snapshots)